api-docs/cloud-dedicated/ref.yml

```diff
@@ -1581,24 +1595,29 @@ paths:
       responses:
         "204":
           description: |
-            Success. Data is written and queryable.
+            Success. All points in the batch are written and queryable.
         "400":
```

Comment:
201 when partial-writes are on, and some lines were written.
Otherwise, still a 400.